File: wildfire-front/src/app/models/position.ts
import {Model} from "./interface/model";

/**
 * Model for position
 */
export class Position implements Model{

  /**
   * Constructor
   */
  constructor() {}

  /**
   * Position in horizontal axis
   */
  posX: number=0;

  /**
   * Position in vertical axis
   */
  posY: number=0;

  /**
   * Method to hydrate object from json
   * @param json Json object
   */
  fromJSON(json: any): any {
    this.posX=json.posX;
    this.posY=json.posY;
  }
}

File: wildfire-api/src/main/java/com/project/wildfire/Models/Position.java
package com.project.wildfire.Models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * Class for position
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class Position {

    /**
     * Position in the horizontal axis
     */
    private int posX;

    /**
     * Position in the vertical axis
     */
    private int posY;
}
